be/src/Infrastructure/Localization/Foundation.Localization/Resources/ClientResources.pl.resx

```diff
@@ -318,4 +318,46 @@
   <data name="NewClientAddress" xml:space="preserve">
     <value>Dodaj adres</value>
   </data>
+  <data name="FieldDefinitionNotFound" xml:space="preserve">
+    <value>Nie znaleziono definicji pola</value>
+  </data>
+  <data name="FieldDefinitionSelectTypeConflict" xml:space="preserve">
+    <value>Nie można przypisać opcji do definicji pola ze względu na wybrany typ</value>
+  </data>
+  <data name="FieldOptionNotFound" xml:space="preserve">
+    <value>Nie znaleziono opcji</value>
+  </data>
+  <data name="BackToField" xml:space="preserve">
+    <value>Powróc do pola</value>
+  </data>
+  <data name="BackToFields" xml:space="preserve">
+    <value>Powróc do pól klienta</value>
```

Comment:
**suggestion:** there are typos and unclear meaning for end user. Please think of better names and fix typos

Reply:
Hi @dawiddworak88, any suggestion? Back to client cards?